test/integration/framework.go

```diff
@@ -110,7 +110,7 @@ func withConfigGetFreshApiserverAndClient(
 
 		if err := server.RunServer(options, stopCh); err != nil {
 			close(serverFailed)
-			t.Fatalf("Error in bringing up the server: %v", err)
+			t.Logf("Error in bringing up the server: %v", err)
```

Comment:
see https://github.com/golang/go/issues/15976
The integration test was failing with data race